client-dotnet/test/EvoMaster.Instrumentation.Tests/Coverage/MethodReplacement/DistanceHelperTest.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using Xunit;
+
+namespace EvoMaster.Instrumentation.Tests.Coverage.MethodReplacement{
+    public class DistanceHelperTest{
+        
+        // Comment this out due to failing on CI
```

Comment:
test cases that for some reasons need to be disabled, should not be commented out, but rather "disabled".
in Java, can be done with `@Disabled`.
need to find equivalent in C#.
the reason is that they ll still show up when running all tests, although will be marked as skipped

Reply:
could it fail on CI due to different locales?

Reply:
Yes, that's the case. Now it's skipped.